### DISCUSSION TITLE
Bug: Code blocks are not rendered properly with custom attributes

### DISCUSSION THREAD

**Discussion by ElBe-Plaq** [mkdocs/mkdocs#3253] [Q&A] [answered]
**Describe the bug**
When using attributes for code blocks (`attribute=1`), code blocks will be rendered as text. 

**To Reproduce**

1. Open a markdown file.
2. Add a code block:
```
`ㅤ``lang attribute=1
Hello world!
`ㅤ``
```
3. Build.
4. Navigate to the file inside of your build.
5. See the error.

**Expected behavior**
Code block should render normally.

**Screenshots**
![image](https://github.com/mkdocs/mkdocs/assets/90863907/1e1b37e6-fad7-4aad-9975-751eb3277657)

**Code**
`mkdocs.yml`:
```yaml
site_name: My Docs

```

`docs\index.md`
```md
# Example

## Working

`ㅤ``py
print("Hello world!")
`ㅤ``

## Broken

`ㅤ``py title="hello.py"
print("Hello world!")
`ㅤ``

```

**Real world example**
Working with title: https://i-language-rust.readthedocs.io/en/latest/
Not working with title: https://i-language-rust.readthedocs.io/en/v1.0.0-alpha3/

**Version**
Mkdocs: 1.4.3

**Comment by pawamoy** [comment]
Markdown tip: use four backticks to show triple-backticks in Markdown code blocks:

`````
Code block showing Markdown:

````
Look at this Python code:

```python
raise SystemExit(0)
```
````
`````

(I'll try to help about your issue in another answer)

**Comment by pawamoy** [comment, marked as answer]
The `title` attribute is only supported by Material for MkDocs I think. Also attribute values must be wrapped in quotes.

**Reply by ElBe-Plaq** [reply]
The issue still appears when using material. Still, thank you for your help. I'll report this issue on the material issue page.

**Comment by Andre601** [comment]
I assume you're using the default highlighter from Python markdown?
Material for MkDocs works best with highlight from PyMDownX which afaik provides the title support you want.

More info: https://squidfunk.github.io/mkdocs-material/reference/code-blocks/

**Comment by facelessuser** [comment]
I'll state this.

1. I cannot see your config, so I do not know if you've set it up properly.
2. The examples you've provided have odd white space between some of your `` ` `` characters and will not work with those present.
3. If you are having issues with parsing of Markdown, it is likely that MkDocs Material is not the appropriate place to file the issue, it would be Pymdown Extensions (which I maintain). As @Andre601 pointed out, the title feature is provided through `pymdownx.highlight` and works if used properly. I am more than happy to provide guidance if sufficient info is provided, but should be done over on my repo to reduce noise here: https://github.com/facelessuser/pymdown-extensions.

**Reply by ElBe-Plaq** [reply]
The white spaces are invisible Unicode characters, for me to be able to show you the file contents. My config can be found [here](https://github.com/I-Language-Development/I-language-rust/blob/v1.0.0-alpha3/Docs/.mkdocs.yml).

**Reply by facelessuser** [reply]
> The white spaces are invisible Unicode characters, for me to be able to show you the file contents.

You can do something like this in the future:

`````
````
```
code
```
````
`````

> My config can be found [here](https://github.com/I-Language-Development/I-language-rust/blob/v1.0.0-alpha3/Docs/.mkdocs.yml).

You need to provide a minimal, reproducible example. I don't see SuperFences in the config at all, and that is required.

**Reply by ElBe-Plaq** [reply]
Okay, I remember what's the issue. It's on my side and can be fixed easily. Thank you!
